doc/source/reference/api/api.rst

```diff
@@ -157,27 +157,16 @@ These objects generate an "index" into multiresolution data.
 Units
 -----
 
-These classes and functions enable yt's symbolic unit handling system.
+yt's symbolic unit handling system is now based on the external library unyt. In
+complement, Dataset objects support the following methods to build arrays and
+scalars with physical dimensions.
 
 .. autosummary::
 
    yt.data_objects.static_output.Dataset.arr
    yt.data_objects.static_output.Dataset.quan
-   ~yt.units.unit_object.define_unit
-   ~yt.units.unit_object.Unit
-   ~yt.units.unit_registry.UnitRegistry
-   ~yt.units.unit_systems.UnitSystem
-   ~yt.units.yt_array.YTArray
-   ~yt.units.yt_array.YTQuantity
-   ~yt.units.yt_array.uconcatenate
-   ~yt.units.yt_array.uintersect1d
-   ~yt.units.yt_array.uunion1d
-   ~yt.units.yt_array.unorm
-   ~yt.units.yt_array.udot
-   ~yt.units.yt_array.uvstack
-   ~yt.units.yt_array.uhstack
-   ~yt.units.yt_array.ustack
-   ~yt.units.yt_array.display_ytarray
```

Comment:
Isn't `display_ytarray` still around somewhere?

We may actually want to use intersphinx to point to the unyt definitions of those of these that still exist.  I'm thinking explicitly of the uconcat, uhstack, etc, which are necessary for combining unyt arrays.

Reply:
sounds good, but I'm barely managing to fix simple rst errors, honestly I don't really want to invest time in learning how to use intersphinx for now :/

Reply:
I don't mind taking this on.

Reply:
thanks !